include/triton/Analysis/Utility.h

```diff
@@ -250,6 +250,9 @@ bool matchMmaV3AndDotOperandLayout(RankedTensorType srcTy,
 bool matchMFMAAndDotOperandShuffleCase(RankedTensorType srcTy,
                                        RankedTensorType dstTy);
 
+bool matchMFMAAndLinearLayoutCase(RankedTensorType srcTy,
```

Comment:
The name here is pretty generic. Can we be specific to use like something `matchMFMAToLinearPermlaneSwapCase`?

Also I think this function can just be in `ConvertLayoutOpToLLVM.cpp` given it's only used there?